Source/ARTDataQuery.m

```diff
@@ -54,7 +54,7 @@ - (NSMutableArray *)asQueryItems {
     NSMutableArray *items = [super asQueryItems];
     if (self.untilAttach) {
         NSAssert(self.realtimeChannel, @"ARTRealtimeHistoryQuery used from outside ARTRealtimeChannel.history");
-        if (self.realtimeChannel.state != ARTRealtimeChannelAttached) {
+        if (self.realtimeChannel.state != ARTChannelStateAttached) {
```

Comment:
If this has changed to `ARTChannelStateAttached` from `ARTRealtimeChannelAttached` is breaking.  If you think that is necessary then please state your case.  However, if you do this, then there are lots of other changes that need doing I reckon, even consider the line above which is ARTRealtimeChannel.history.  Then would `ARTRealtimeClosed ` becomes `ARTConnectionClosed` for example. Why did you change it, what's the thinking?